src/Raven.Client/Documents/Session/AsyncSessionDocumentTimeSeries.cs

```diff
@@ -31,10 +32,20 @@ public AsyncSessionDocumentTimeSeries(InMemoryDocumentSessionOperations session,
 
         public Task<TimeSeriesEntry[]> GetAsync(DateTime? from = null, DateTime? to = null, int start = 0, int pageSize = int.MaxValue, CancellationToken token = default)
         {
-            return GetAsyncInternal<TimeSeriesEntry>(from, to, start, pageSize, token);
+            return GetTimeSeriesAndIncludes<TimeSeriesEntry>(from, to, includes : null, start, pageSize, token);
+        }
+
+        public Task<TimeSeriesEntry[]> GetAsync(DateTime? from, DateTime? to, Action<ITimeSeriesIncludeBuilder> includes, int start = 0, int pageSize = int.MaxValue, CancellationToken token = default)
+        {
+            return GetTimeSeriesAndIncludes<TimeSeriesEntry>(from, to, includes, start, pageSize, token);
         }
 
         public async Task<TTValues[]> GetAsyncInternal<TTValues>(DateTime? from = null, DateTime? to = null, int start = 0, int pageSize = int.MaxValue, CancellationToken token = default) where TTValues : TimeSeriesEntry
+        {
+            return await GetTimeSeriesAndIncludes<TTValues>(from, to, includes: null, start, pageSize, token).ConfigureAwait(false);
```

Comment:
no need to await, you can return the task, right?

Reply:
right. done